Havit/Havit/Screens/CategoryContents/View/Components/Cells/MorePanModalTableViewCell.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MoreBottomSheetTableViewCell.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/15.
+//
+
+import UIKit
+
+class MorePanModalTableViewCell: BaseTableViewCell {
+
+    // MARK: - Properties
```

Comment:
p1;

주석 
```suggestion
    // MARK: - property
```

이걸로 맞추기로 했어요!

Reply:
f7243580